.github/workflows/static-analysis.yml

```diff
@@ -5,13 +5,13 @@ on:
   pull_request:
     branches:
       - "*.x"
-      - "master"
   push:
     branches:
       - "*.x"
-      - "master"
 
 jobs:
   static-analysis:
     name: "Static Analysis"
-    uses: "greg0ire/.github/.github/workflows/static-analysis.yml@1.1.1"
+    uses: "doctrine/.github/.github/workflows/static-analysis.yml@1.4.0"
+    with:
+      php-version: "7.4"
```

Comment:
We could remove some files from PHPStan's exclusion list if we run analysis with PHP 8.1 :)

Reply:
Yes, but then again, I'm not eager to analyze fixtures for the Proxy tests. 😉